src/database/models.py

```diff
@@ -259,25 +260,40 @@ class Media(models.Model):
     is_deleted = models.BooleanField(default=False, blank=True)
     order = models.PositiveIntegerField(default=0, blank=True, null=True)
     tags = models.ManyToManyField(Tag, related_name="media_tags", blank=True)
+    more_info = models.JSONField(blank=True, null=True)
 
     def __str__(self):
         return str(self.id) + "-" + self.name + "(" + self.file.name + ")"
 
     def simple_json(self):
-        return {
+        obj=  {
             "id": self.id,
             "name": self.name,
             "url": self.file.url,
         }
 
+        if hasattr(self, "user_upload"): 
+            obj["created_at"] = self.user_upload.created_at
```

Comment:
Do we only want the created_at for user uploads?  If it is not an upload could you determine it from a file date?

Reply:
Yes, currently "updated_at" field isnt used in any use case yet. Only you have some new suggestions/use cases?  Also I am not sure I understand what you mean by "If it is not an upload could you determine it from a file date?"